tests/drivers/sensor/accel/src/main.c

```diff
@@ -11,72 +11,312 @@
  * @}
  */
 
-#include <zephyr/ztest.h>
+#include <zephyr/drivers/emul.h>
 #include <zephyr/drivers/sensor.h>
+#include <zephyr/emul/bmi160.h>
+#include <zephyr/math/util.h>
+#include <zephyr/ztest.h>
+
+#include <bmi160.h>
 
 struct sensor_accel_fixture {
-	const struct device *accel_spi;
-	const struct device *accel_i2c;
+	const struct device *accel_0;
```

Comment:
Nit: The previous names made it more obvious why we were running the same test for two different sensor nodes.